src/policy/policy.h

```diff
@@ -14,14 +14,21 @@
 
 class CCoinsViewCache;
 
+/** Recommended transaction fee by Dogecoin Core developers
+  *
+  * All fee defaults used throughout the client derive their
+  * value from this base default.
+  */
+static const CAmount RECOMMENDED_MIN_TX_FEE = COIN / 100;
```

Comment:
This works, but does break the node/wallet isolation (which is why I infer this wasn't here in the first place).

I think that's fine, just wanting to make sure that's an intentional decision.

Reply:
The recommended fee is a recommendation not only for the wallet, but also for mining inclusion (`DEFAULT_BLOCK_MIN_TX_FEE`) and the dogecoin-specific dust limit (`DEFAULT_DUST_LIMIT`), so arguably the recommendation is policy, the wallet implementation then of course follows the recommendation by default.

Reply:
To clarify further, I think referencing policy from the wallet is fine, we (used to) do that for "smartfees" too. Doing it the other way around, though, like including wallet.h (or even validation.h for that matter) into policy files would break the "isolation" and generally would be bad architecture. 

I had toyed with the idea of creating an additional header file `policy/fee-base.h` but decided that making a header just for a single constant is a bit overkill, so I put it with the rest of the generic policy constants. Can change that if you like.